packages/playwright-test/src/runner.ts

```diff
@@ -609,7 +634,8 @@ function filterByFocusedLine(suite: Suite, focusedTestFileLines: TestFileFilter[
   const suiteFilter = (suite: Suite) => {
     return !!suite.location && testFileLineMatches(suite.location.file, suite.location.line, suite.location.column);
   };
-  const testFilter = (test: TestCase) => testFileLineMatches(test.location.file, test.location.line, test.location.column);
+  // Project setup files are always included.
+  const testFilter = (test: TestCase) => setupFiles.has(test.location.file) || testFileLineMatches(test.location.file, test.location.line, test.location.column);
```

Comment:
```suggestion
  const testFilter = (test: TestCase) => setupFiles.has(test._requireFile) || testFileLineMatches(test.location.file, test.location.line, test.location.column);
```

But perhaps, we should introduce `_fileType: 'setup' | 'file'` next to `_requireFile`?

Reply:
Changed to _requireFile for now as we look up in the set once for TestCase._requireFile and once for TestGroup.requireFile. We can unify it later.